zenoh-java/src/commonMain/kotlin/io/zenoh/query/Get.kt

```diff
@@ -14,181 +14,25 @@
-        fun consolidation(consolidation: ConsolidationMode): Builder<R> {
-            this.consolidation = consolidation
-            return this
-        }
-
-        /** Specify the timeout. */
-        fun timeout(timeout: Duration): Builder<R> {
-            this.timeout = timeout
-            return this
-        }
-
-        /**
-         * Specify a string value. A [Value] is generated with the provided message, therefore
-         * this method is equivalent to calling `withValue(Value(message))`.
-         */
-        fun withValue(message: String): Builder<R> {
-            this.value = Value(message)
-            return this
-        }
-
-        /** Specify a [Value]. */
-        fun withValue(value: Value): Builder<R> {
-            this.value = value
-            return this
-        }
-
-        /** Specify an attachment. */
-        fun withAttachment(attachment: ByteArray): Builder<R> {
-            this.attachment = attachment
-            return this
-        }
-
-        /**
-         * Specify an action to be invoked when the Get operation is over.
-         *
-         * Zenoh will trigger ths specified action once no more replies are to be expected.
-         */
-        fun onClose(action: () -> Unit): Builder<R> {
-            this.onClose = action
-            return this
-        }
-
-        /** Specify a [Callback]. Overrides any previously specified callback or handler. */
-        fun with(callback: Callback<Reply>): Builder<Unit> = Builder(this, callback)
-
-        /** Specify a [Handler]. Overrides any previously specified callback or handler. */
-        fun <R2> with(handler: Handler<Reply, R2>): Builder<R2> = Builder(this, handler)
-
-        /** Specify a [BlockingQueue]. Overrides any previously specified callback or handler. */
-        fun with(blockingQueue: BlockingQueue<Optional<Reply>>): Builder<BlockingQueue<Optional<Reply>>> = Builder(this, BlockingQueueHandler(blockingQueue))
-
-        /**
-         * Resolve the builder triggering the query.
-         *
-         * @return The receiver [R] from the specified [Handler] (if specified).
-         */
-        @Throws(ZenohException::class)
-        fun res(): R? {
-            require(callback != null || handler != null) { "Either a callback or a handler must be provided." }
-            val resolvedCallback = callback ?: Callback { t: Reply -> handler?.handle(t) }
-            val resolvedOnClose = fun() {
-                onClose?.invoke()
-                handler?.onClose()
-            }
-            return session.run {
-                resolveGet(
-                    selector,
-                    resolvedCallback,
-                    resolvedOnClose,
-                    handler?.receiver(),
-                    timeout,
-                    target,
-                    consolidation,
-                    value,
-                    attachment
-                )
-            }
-        }
-    }
-}
+data class GetOptions(
```

Comment:
There are no getters/setters in GetOptions. Corresponding example also doesn't demonstrate `GetOptions` usage

Reply:
Ok I'll add the demonstration to the Get example, but we don't need getters and setters for data classes, Kotlin implements them for you. The reason you see some getters and setters in other data classes is for the QoS parameters, just as a utility.